src/components/ContentPreview/ContentPreview.js

```diff
@@ -228,6 +235,13 @@ class ContentPreview extends PureComponent<Props, State> {
             currentFileId: fileId,
             prevFileIdProp: fileId,
         };
+        this.props.onMetric(
+            METRIC_TYPE_ELEMENTS_LOAD_METRIC,
```

Comment:
Can we standardize on `load` or `js_ready`? It looks like we're using them to mean the same thing. It seems like `load` makes the most sense. What do you think?